docs/form/page.md

```diff
@@ -10,43 +10,44 @@ Pages sit below the [`Subsection`](subsection) level of a form definition.
 
 An example page might look something like this:
 
-```json
-"property_postcode": {
-  "header": "",
-  "description": "",
-  "questions": {
-    ...
-  },
-  "depends_on": [
-    {
-      "needstype": 1
+```
+class Form::Sales::Pages::PropertyPostcode < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = property_postcode
+    @depends_on = [{ "needstype" => 1 }]
+    @title_text = {
+      "translation": "translation1",
+      "arguments": [
+        {
+          "key": "some_general_field",
+          "label": true,
+          "i18n_template": "template1"
+        }
+      ]
     }
-  ],
-  "title_text": {
-    "translation": "translation1",
-    "arguments": [
-      {
-        "key": "some_general_field",
-        "label": true,
-        "i18n_template": "template1"
-      }
-    ]
-  },
-  "informative_text": {
-    "translation": "translation2",
-    "arguments": [
-      {
-        "key": "some_currency_method",
-        "label": false,
-        "i18n_template": "template2",
-        "currency": true,
-      }
+    @informative_text": {
+      "translation": "translation2",
+      "arguments": [
+        {
+          "key": "some_currency_method",
+          "label": false,
+          "i18n_template": "template2",
+          "currency": true,
+        }
+      ]
+    }
+  end
+
+  def questions
+    @questions ||= [
+      Form::Sales::Questions::Question.new(nil, nil, self),
     ]
-  },
-}
+  end
+end
 ```
 
-In the above example the the subsection has the id `property_postcode`. This id is used for the url of the web page, but the underscore is replaced with a hash, so the url for this page would be `[environment-url]/logs/[log-id]/property-postcode` e.g. on staging this url might look like the following: `https://dluhc-core-staging.london.cloudapps.digital/logs/1234/property-postcode`.
+In the above example the the subsection has the id `property_postcode`. This id is used for the url of the web page, but the underscore is replaced with a hash, so the url for this page would be `[environment-url]/logs/[log-id]/property-postcode` e.g. on staging this url might look like the following: `https://staging.submit-social-housing-data.communities.gov.uk/logs/1234/property-postcode`.
```

Comment:
Do you mean dash instead of hash? in: "but the underscore is replaced with a hash"

Reply:
yep, good spot!